src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
All packages in this project should be subpackages of org.trustedanalytics.h2oscoringengine.